app/views/documentation/index.html.erb

```diff
@@ -74,5 +86,23 @@ production:
       <dt>How does SlashDeploy know who I am on GitHub?</dt>
       <dd>When you first run the /deploy command, we ask you to authenticate with your GitHub account, which is then linked to your Slack user account.</dd>
     </dl>
+
+    <hr />
+
+    <h5 id="troubleshooting">Troubleshooting</h5>
+    <p>Here we have some debugging information about the common issues and how to resolve them.
+    <dl>
+      <dt id="error-1">Error 1: Deployment did not start</dt>
+      <dd>If a deployment doesn't start with in 30 seconds, you will get a Slack notification letting you know something went wrong. This might happen for a number of reasons, so check the following.
```

Comment:
Small typo: `/with in/within`

Reply:
Also, maybe replace `30` with `<%= time_ago_in_words(GithubDeploymentWatchdogWorker::DEFAULT_DELAY.from_now, include_seconds: true) %>`